cirq-core/cirq/circuits/qasm_output_test.py

```diff
@@ -191,6 +191,24 @@ def test_h_gate_with_parameter():
     )
 
 
+def test_qasm_global_pahse():
+    (q0,) = _make_qubits(1)
```

Comment:
please change this test into testing a use case. the example from the original issue should suffice

```py3
def test_qasm_global_pahse():
    c = cirq.Circuit([cirq.global_phase_operation(np.exp(1j * 5))])
    assert cirq.qasm(c) == ... 
```

Reply:
Thank you for comment.
I changed test case for suitable issue. An if statement is added to `qasm_output.py` to maintain compatibility of the output string with other cases. If this branch was not included, the output of a circuit consisting only of global phase gates would look like this. In this case, the global phase gate has no qubit to apply, but two newlines are added at the end because it is recognized as one of the operations.

Any comments on which representation is more appropriate would be appreciated.

```
"""OPENQASM 2.0;
include "qelib1.inc";


// Qubits: []


"""
```